asQ/diag_preconditioner.py

```diff
@@ -61,6 +59,15 @@ class DiagFFTPC(TimePartitionMixin):
     'diagfft_alpha': <float>
         The circulant parameter to use in the preconditioning matrix.
         Defaults to 1e-3.
+
+    'diagfft_complex_proxy': <'vector', 'mixed'>
+        Which implementation of the complex-proxy module to use.
+        Default is 'vector'.
+
+        'vector': the real-imag components of the complex function space are implemented
+            as a 2-component VectorFunctionSpace
+        'mixed': the real-imag components of the complex function space are implemented
+            as a 2-component MixedFunctionSpace
```

Comment:
do these options get pulled into the documentation somewhere?

Reply:
We don't have a central documentation document anywhere yet (we should do that soon though), but these options will come up with `pydoc`.